test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -28,7 +28,7 @@ public class TestClient : IDisposable, IInvocationBinder
         public Channel<byte[]> Application { get; }
         public Task Connected => ((TaskCompletionSource<bool>)Connection.Metadata["ConnectedTask"]).Task;
 
-        public TestClient(bool synchronousCallbacks = false)
+        public TestClient(bool synchronousCallbacks = false, bool shouldHaveId = false)
```

Comment:
consider renaming to`shouldHaveId` to  `addClaimId` - as it is now it is not clear what the id refers to